.editorconfig

```diff
@@ -20,4 +20,4 @@ indent_size = 2
 trim_trailing_whitespace = false
```

Comment:
Once we are around, I believe using whitespace at the end of MD lines is tricky and easy to miss. It's better to avoid using this MD feature and prefer `<br />` or new paragraph instead. (It's rarely needed anyway, at least according to my experience.)

If you agree, we could end up with a neat and simple default `.editorconfig` 🙂.

Reply:
Maybe let's skip it and focus on default `.editorconfig` in https://github.com/lmc-eu/code-quality-tools.